assets/src/edit-story/elements/text/index.js

```diff
@@ -36,15 +36,10 @@ export const defaultAttributes = {
   bold: false,
   fontFamily: 'Roboto',
   fontFallback: ['Helvetica Neue', 'Helvetica', 'sans-serif'],
-  fontWeight: 400,
```

Comment:
Ah, more ore note: currently `Story_Post_Type(.php)` is using `fontWeight` for loading the correct font in the FE. This will be moved to client-side with #1273 (cc @swissspidy) but something to consider anyway for font generation.

Reply:
@miina, is this fixed or does it need follow-up?

Reply:
This needs a follow-up.
For testing:
1. Create a Text block
2. Assign Mukta Mahee font, Extra Light, toggle Italic
3. Save and preview
4. See the font loaded in the FE: https://fonts.googleapis.com/css2?display=swap&family=Mukta+Mahee which includes only `normal` font-style, it should, however, put together correct font link to fetch the correct font variants: italic + extra light in this case (variant `[1, 200]`).

We actually have an issue which should include a fix for that as well: https://github.com/google/web-stories-wp/issues/923

The issue was created thinking of the previous draftjs bold/italic option only, however, now it should fix it for the full text element -- it's the same logic needed there anyway -- partial selection or full.

That issue is already in the Sprint backlog, I might take a look at it meanwhile based on your PR.

Reply:
I believe this is being addressed by @obetomuniz in https://github.com/google/web-stories-wp/pull/1275